### PR TITLE
feat($controller): `$controller` in depth

### LINKED ISSUE #19: $controller in depth

### PR DESCRIPTION
**Pull request by alexlrobertson**
Closes #19

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -535,6 +536,18 @@ it('does something async', function (done) {
 });
 ```
 
+## [`$controller`](https://docs.angularjs.org/api/ngMock/service/$controller)
+
+> `$controller` service is responsible for instantiating controllers.
+
+The `$controller` services allows you to instantiate a controller for testing. It requires the name of the controller as a string.
+
+Optionally, you can specify locals, which allow you to add mock dependencies which override the standard provided dependencies. This is helpful for mocking services and injecting dependencies coming from state resolves.
```
Comment (tykowale) [inline README.md]: Possibly throw in what it looks like?

Comment (alexlrobertson) [inline README.md]: :+1:

Comment (alexlrobertson) [inline README.md]: :eyes:

**Comment by tykowale** [comment]
:cake:

**Pull request merged**